frontend/src/components/InfoDropDown/InfoDropDownOption.tsx

```diff
@@ -0,0 +1,41 @@
+import styled, { css } from 'styled-components';
+import useBooleanState from '~/hooks/useBooleanState';
+
+interface InfoDropDownOptionProps {
+  value: string;
+  onClick: React.MouseEventHandler;
+}
+
+function InfoDropDownOption({ value, onClick }: InfoDropDownOptionProps) {
+  const { value: isHover, setTrue: onHover, setFalse: onNotHover } = useBooleanState(false);
+
+  return (
+    <StyledDropDownOption
+      isHover={isHover}
+      data-name={value}
+      onMouseDown={onClick}
+      onMouseEnter={onHover}
+      onMouseLeave={onNotHover}
```

Comment:
```suggestion
      onMouseDown={mouseDown}
      onMouseEnter={mouseEnter}
      onMouseLeave={mouseLeave}
```
이런식으로 네이밍을 맞춰주면 좋을 것 같아요